mongo/collection.go

```diff
@@ -545,7 +592,18 @@ func (coll *Collection) UpdateOne(ctx context.Context, filter interface{}, updat
 	return coll.updateOrReplace(ctx, f, update, false, rrOne, true, opts...)
 }
 
-// UpdateMany updates multiple documents in the collection.
+// UpdateMany executes an update operation (https://docs.mongodb.com/manual/reference/command/update/) to update
+// documents in the collection.
+//
+// The filter parameter must be a document containing query operators and can be used to select the documents to be
+// updated. It cannot be nil. An empty document (e.g. bson.D{}) should be used to select all documents. If the filter does
```

Comment:
Why point out specifically about an empty document?  It's so uncommon to want to match every document.

Reply:
I pointed out empty document here and in other places because the wording at the beginning is "must be a document containing query operators" so I think a natural question is "what happens if there are no query operators in the filter". It may be uncommon, but I see value in keeping it.

Reply:
Trying to think like a typical user, I don't agree that it's a natural question for an UpdateMany operation.  Perhaps for Find, but not UpdateMany.  If you want to be more precise, you can change the first sentence to "zero or more query operators", but I don't think that's necessary either.

Reply:
That's fair. I removed that line from the UpdateMany method but left it for methods like Find.